packages/conf-haveged/conf-havaged.1.0.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+homepage: "https://linux.die.net/man/8/haveged"
+author: "A. Seznec and N. Sendrier"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-1.0-or-later"
+depexts: [
```

Comment:
```suggestion
build: ["haveged" "--help"]
depexts: [
```
Maybe that could help detect if it is actually installed quicker

Reply:
Or maybe not, it seems installed in /usr/sbin on debian which might be inaccessible for non-privileged users